src/core/components/layout/components/container/container.tsx

```diff
@@ -1,27 +1,49 @@
 import { HTMLAttributes, ReactNode } from 'react';
 import { SerializedStyles } from '@emotion/react';
-import { container, containerBorder } from './styles';
+import {
+	container,
+	containerSideBorders,
+	containerTopBorder,
+	containerBackground,
+} from './styles';
 import { Props } from '@guardian/src-helpers';
+import { border } from '@guardian/src-foundations/palette';
 
 interface Container extends HTMLAttributes<HTMLDivElement>, Props {
-	border?: boolean;
+	sideBorders?: boolean;
+	topBorder?: boolean;
+	backgroundColor?: string;
+	borderColor?: string;
 	cssOverrides?: SerializedStyles | SerializedStyles[];
 	children: ReactNode;
 }
 
 const Container = ({
-	border = false,
+	sideBorders = false,
+	topBorder = false,
+	backgroundColor,
+	borderColor = border.secondary,
```

Comment:
Following on from the previous comment, I'd prefer `border.secondary` to be specified in the styles rather than here. This may be overridden by further styles if `borderColor` is explicitly passed as a prop.